src/jni/org/netbsd/liblpm/LPM.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2016 Henry Rodrick <henry at holodisc org uk>
+ * All rights reserved.
+ *
+ * Use is subject to license terms, as specified in the LICENSE file.
+ */
+
+package org.netbsd.liblpm;
+
+import java.io.*;
+
+public class LPM<T> {
```

Comment:
It would also be nice to support IP address/network in bytes, e.g. users might have `java.net.InetAddress` (and, obviously, they would like to use `getAddress()` then).